### PR TITLE
Static content generation for podcasts and videos

### PR DESCRIPTION
**Pull request by jorgelainfiesta**

### CODE REVIEW

File: app/adapters/application.js [hunk 1]
```diff
@@ -1,16 +1,9 @@
-// import Ember from 'ember';
-// import FirebaseAdapter from 'emberfire/adapters/firebase';
+import DS from 'ember-data';
 
-// const { inject } = Ember;
+export default DS.JSONAPIAdapter.extend(DS.BuildURLMixin, {
```
Comment (taras) [inline app/adapters/application.js]: with the new version of `ember-writer`, you only need
`export { default } from 'ember-writer/adapters/application';`

File: app/models/author.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import Model from 'ember-data/model';
```
Comment (taras) [inline app/models/author.js]: With the new version of `ember-writer`, you can do this:
```js
import Author from 'ember-writer/models/author';

export default Author.extend({
  twitter: attr(),
  github: attr()
});
```

File: app/models/post.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+
+export default Model.extend({
```
Comment (taras) [inline app/models/post.js]: With the new version of `ember-writer` you can do this:

```js
import Post from 'ember-writer/models/post';

export default Post.extend({
  mediaId: attr(),
  kind: attr(),
  section: attr(),
  imageUrl: attr()
});
```

File: app/models/tag.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+
+export default Model.extend({
```
Comment (taras) [inline app/models/tag.js]: You can delete this file with the new version of ember-writer

File: app/routes/application.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+const { RSVP: { hash } } = Ember;
+
+export default Ember.Route.extend({
+  model() {
+    return hash({
```
Comment (taras) [inline app/routes/application.js:6]: You don't need to fetch everything anymore. You can just fetch exactly what you need for each route.

**Review by taras — CHANGES_REQUESTED** [review]
Let's tag `ember-writer` to sha 81a96647b051658e808da0436a434f9698882edd

It'll require a few updates here.

File: app/routes/application.js (hunk 5 above)
Comment (jorgelainfiesta) [reply app/routes/application.js:6]: Is it possible to query data?

Comment (jorgelainfiesta) [reply app/routes/application.js:6]: It seems not, but that's ok. Everything works like a charm :)

Comment (taras) [reply app/routes/application.js:6]: right, we can't query but we can filter which should be enough.

**Comment by taras** [comment]
@jorgelainfiesta is this all ready to go?

**Comment by jorgelainfiesta** [comment]
Yep

**Pull request merged**